src/test/java/seedu/address/logic/parser/MarkTaskCommandParserTest.java

```diff
@@ -3,7 +3,7 @@
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_TASK;
```

Comment:
Good that you define / use a new static value instead of using that of the `Person`'s